tests/framework/helpers/HtmlTest.php

```diff
@@ -671,11 +670,11 @@ public function testRemoveCssClass()
     {
         $options = ['class' => 'test test2 test3'];
         Html::removeCssClass($options, 'test2');
-        $this->assertEquals(['class' => 'test test3'], $options);
+        $this->assertEquals(['class' => ['test', 2 => 'test3']], $options);
```

Comment:
@samdark  I was checking the doc and it seems like $options['class'] can be an array or an string depending on the input. http://www.yiiframework.com/doc-2.0/yii-helpers-basehtml.html#addCssClass()-detail basically if it started as a non-empty array, then it will remain as a non-empty array after being modified.

if it started as string then it will remain as string after modifications.